task3/slices/slices_test.go

```diff
@@ -0,0 +1,51 @@
+package slices
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestMaxMainCases(t *testing.T) {
+	var tests = []struct {
+		arr       []string
+		maxString string
+	}{
+		{[]string{"a", "bb", "ccc"}, "ccc"},
+		{[]string{"a", "b", "c"}, "a"},
+		{[]string{"aaa", "b"}, "aaa"},
+		{[]string{""}, ""},
+	}
+	for _, c := range tests {
+		got, _ := Max(c.arr)
+		if got != c.maxString {
+			t.Errorf("Max(%v) == %s, want %s", c.arr, got, c.maxString)
+		}
+	}
+}
+
+func TestMaxEmptyArray(t *testing.T) {
+	defer func() {
```

Comment:
Usually you don't want defers in `Test*` functions.
We'll take a look at a package that makes testing easier next week.